internal/cli/kraft/cloud/certificate/remove/remove.go

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2022, Unikraft GmbH and The KraftKit Authors.
```

Comment:
```suggestion
// Copyright (c) 2024, Unikraft GmbH and The KraftKit Authors.
```

This but everywhere.